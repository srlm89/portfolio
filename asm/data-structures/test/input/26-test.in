ERASE actual/26-test.out
INSERT 20 2.370776 The Mummy 1-2   1080p
PRINT_IN_ORDER actual/26-test.out
INSERT 2 5.882356 Hellboy 2     1080p
INSERT 3 0.612648 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p]
REMOVE_ID 20
PRINT_IN_ORDER actual/26-test.out
INSERT 15 1.666360 Chicken Run [2000] [720p]
INSERT 21 0.641705 X-Men 3   1080p
INSERT 6 4.309581 Babylon A.D    720p
REMOVE_ID 15
PRINT_IN_ORDER actual/26-test.out
INSERT 9 3.828060 Ong.Bak.2.2008.BluRay.x264-BestHD   720p
INSERT 0 3.769339 Hellboy 2     1080p
INSERT 5 9.830756 Ongbak 2003 720p
INSERT 34 9.818277 Wanted   1080p
REMOVE_ID 6
REMOVE_ID 5
PRINT_IN_ORDER actual/26-test.out
INSERT 13 9.492849 Max Payne    720p
INSERT 16 3.798676 James Bond 01 - Dr. No [1962] [720p]
INSERT 4 0.195422 Bride Wars   720p
INSERT 24 9.327714 Iron Man    1080p
INSERT 23 8.509363 James Bond 09 - The Man with the Golden Gun [1974]
REMOVE_ID 13
REMOVE_ID 0
PRINT_IN_ORDER actual/26-test.out
INSERT 11 8.087893 Pinocchio [1940] [720p]
INSERT 29 3.173707 21   1080p
INSERT 31 9.520840 PBS
INSERT 10 1.464682 Pursuit_Of_Happyness[2006]   720p
INSERT 41 0.414628 Rocky 4 [1985] [BluRay] [720p] [AC3]
INSERT 45 5.741547 Big Nothing   720p
REMOVE_ID 24
REMOVE_ID 45
REMOVE_ID 4
PRINT_IN_ORDER actual/26-test.out
INSERT 33 9.824923 Stargate-The Ark of Truth 720p
INSERT 32 5.382049 Juno 2007 720p
INSERT 48 5.443910 Baby on Board (2009)    720p
INSERT 25 9.420102 Ju-on__The_Grudge_(2003)   720p
INSERT 28 6.642713 Pinocchio [1940] [720p]
INSERT 51 2.482692 Initial D [720p]
INSERT 18 7.545011 Punisher 2   720p
REMOVE_ID 48
REMOVE_ID 28
REMOVE_ID 11
PRINT_IN_ORDER actual/26-test.out
INSERT 50 1.446025 Tomb Raider Collection (1-2)    720p
INSERT 42 4.765930 Interview with the Vampire   720p
INSERT 30 4.170517 Big Stan   1080p
INSERT 54 7.474814 James Bond 21 - Casino Royale [2006]
INSERT 7 3.339416 Land Of the Lost   720p
INSERT 14 8.923610 10,000 BC 720p
INSERT 46 2.829013 Shoot.Em.Up.720p.Bluray.x264-SEPTiC
INSERT 37 7.403041 PIXAR Movie
REMOVE_ID 23
REMOVE_ID 2
REMOVE_ID 34
REMOVE_ID 46
PRINT_IN_ORDER actual/26-test.out
INSERT 61 0.904714 Alien At The Attic 720P
INSERT 49 7.222352 Starship Troopers 2  720p
INSERT 63 9.043006 Hellboy 1    1080p
INSERT 36 8.716279 Final Fantasy VII Advent Children [2005] [1080p]
INSERT 22 4.288028 The Proposal [2009] [720p]
INSERT 17 8.909825 I Am Legend    720p
INSERT 27 7.300815 Casablana.1942.1080p.BluRay.x264.Codres
INSERT 56 0.284166 Punisher Warzone [720p]
INSERT 68 1.686811 Chicken Little 1080p
REMOVE_ID 29
REMOVE_ID 63
REMOVE_ID 16
REMOVE_ID 31
PRINT_IN_ORDER actual/26-test.out
INSERT 59 0.841316 Full.Metal.Jacket.REMASTERED.1987.720p.HDDVD.x264-SEPTiC
INSERT 8 6.639775 James Bond 02 - From Russia With Love [1963]
INSERT 39 4.794048 Golden Compass   720p
INSERT 26 4.911898 Rocky Balboa    720p
INSERT 44 0.161225 Back.to.the.Future.Part.II.1989.720p.HDTV.x264
INSERT 35 1.239642 Van.Helsing  1080p
INSERT 40 9.511793 Cleanwest 720p
INSERT 43 2.709857 Golden Compass   720p
INSERT 55 7.042460 Jeniffer's Body ( 2009)    1080p
INSERT 70 6.373600 5_Centimeters_Per_Second_(2007)_[720p,BluRay,x264]_-_THORA
REMOVE_ID 9
REMOVE_ID 43
REMOVE_ID 30
REMOVE_ID 70
REMOVE_ID 3
PRINT_IN_ORDER actual/26-test.out
