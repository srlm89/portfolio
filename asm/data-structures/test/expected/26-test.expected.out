[ 20 2.370776 The Mummy 1-2   1080p ]
[ 2 5.882356 Hellboy 2     1080p ][ 3 0.612648 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ]
[ 2 5.882356 Hellboy 2     1080p ][ 3 0.612648 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 6 4.309581 Babylon A.D    720p ][ 21 0.641705 X-Men 3   1080p ]
[ 0 3.769339 Hellboy 2     1080p ][ 2 5.882356 Hellboy 2     1080p ][ 3 0.612648 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 9 3.828060 Ong.Bak.2.2008.BluRay.x264-BestHD   720p ][ 21 0.641705 X-Men 3   1080p ][ 34 9.818277 Wanted   1080p ]
[ 2 5.882356 Hellboy 2     1080p ][ 3 0.612648 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 4 0.195422 Bride Wars   720p ][ 9 3.828060 Ong.Bak.2.2008.BluRay.x264-BestHD   720p ][ 16 3.798676 James Bond 01 - Dr. No [1962] [720p] ][ 21 0.641705 X-Men 3   1080p ][ 23 8.509363 James Bond 09 - The Man with the Golden Gun [1974] ][ 24 9.327714 Iron Man    1080p ][ 34 9.818277 Wanted   1080p ]
[ 2 5.882356 Hellboy 2     1080p ][ 3 0.612648 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 9 3.828060 Ong.Bak.2.2008.BluRay.x264-BestHD   720p ][ 10 1.464682 Pursuit_Of_Happyness[2006]   720p ][ 11 8.087893 Pinocchio [1940] [720p] ][ 16 3.798676 James Bond 01 - Dr. No [1962] [720p] ][ 21 0.641705 X-Men 3   1080p ][ 23 8.509363 James Bond 09 - The Man with the Golden Gun [1974] ][ 29 3.173707 21   1080p ][ 31 9.520840 PBS ][ 34 9.818277 Wanted   1080p ][ 41 0.414628 Rocky 4 [1985] [BluRay] [720p] [AC3] ]
[ 2 5.882356 Hellboy 2     1080p ][ 3 0.612648 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 9 3.828060 Ong.Bak.2.2008.BluRay.x264-BestHD   720p ][ 10 1.464682 Pursuit_Of_Happyness[2006]   720p ][ 16 3.798676 James Bond 01 - Dr. No [1962] [720p] ][ 18 7.545011 Punisher 2   720p ][ 21 0.641705 X-Men 3   1080p ][ 23 8.509363 James Bond 09 - The Man with the Golden Gun [1974] ][ 25 9.420102 Ju-on__The_Grudge_(2003)   720p ][ 29 3.173707 21   1080p ][ 31 9.520840 PBS ][ 32 5.382049 Juno 2007 720p ][ 33 9.824923 Stargate-The Ark of Truth 720p ][ 34 9.818277 Wanted   1080p ][ 41 0.414628 Rocky 4 [1985] [BluRay] [720p] [AC3] ][ 51 2.482692 Initial D [720p] ]
[ 3 0.612648 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 7 3.339416 Land Of the Lost   720p ][ 9 3.828060 Ong.Bak.2.2008.BluRay.x264-BestHD   720p ][ 10 1.464682 Pursuit_Of_Happyness[2006]   720p ][ 14 8.923610 10,000 BC 720p ][ 16 3.798676 James Bond 01 - Dr. No [1962] [720p] ][ 18 7.545011 Punisher 2   720p ][ 21 0.641705 X-Men 3   1080p ][ 25 9.420102 Ju-on__The_Grudge_(2003)   720p ][ 29 3.173707 21   1080p ][ 30 4.170517 Big Stan   1080p ][ 31 9.520840 PBS ][ 32 5.382049 Juno 2007 720p ][ 33 9.824923 Stargate-The Ark of Truth 720p ][ 37 7.403041 PIXAR Movie ][ 41 0.414628 Rocky 4 [1985] [BluRay] [720p] [AC3] ][ 42 4.765930 Interview with the Vampire   720p ][ 50 1.446025 Tomb Raider Collection (1-2)    720p ][ 51 2.482692 Initial D [720p] ][ 54 7.474814 James Bond 21 - Casino Royale [2006] ]
[ 3 0.612648 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 7 3.339416 Land Of the Lost   720p ][ 9 3.828060 Ong.Bak.2.2008.BluRay.x264-BestHD   720p ][ 10 1.464682 Pursuit_Of_Happyness[2006]   720p ][ 14 8.923610 10,000 BC 720p ][ 17 8.909825 I Am Legend    720p ][ 18 7.545011 Punisher 2   720p ][ 21 0.641705 X-Men 3   1080p ][ 22 4.288028 The Proposal [2009] [720p] ][ 25 9.420102 Ju-on__The_Grudge_(2003)   720p ][ 27 7.300815 Casablana.1942.1080p.BluRay.x264.Codres ][ 30 4.170517 Big Stan   1080p ][ 32 5.382049 Juno 2007 720p ][ 33 9.824923 Stargate-The Ark of Truth 720p ][ 36 8.716279 Final Fantasy VII Advent Children [2005] [1080p] ][ 37 7.403041 PIXAR Movie ][ 41 0.414628 Rocky 4 [1985] [BluRay] [720p] [AC3] ][ 42 4.765930 Interview with the Vampire   720p ][ 49 7.222352 Starship Troopers 2  720p ][ 50 1.446025 Tomb Raider Collection (1-2)    720p ][ 51 2.482692 Initial D [720p] ][ 54 7.474814 James Bond 21 - Casino Royale [2006] ][ 56 0.284166 Punisher Warzone [720p] ][ 61 0.904714 Alien At The Attic 720P ][ 68 1.686811 Chicken Little 1080p ]
[ 7 3.339416 Land Of the Lost   720p ][ 8 6.639775 James Bond 02 - From Russia With Love [1963] ][ 10 1.464682 Pursuit_Of_Happyness[2006]   720p ][ 14 8.923610 10,000 BC 720p ][ 17 8.909825 I Am Legend    720p ][ 18 7.545011 Punisher 2   720p ][ 21 0.641705 X-Men 3   1080p ][ 22 4.288028 The Proposal [2009] [720p] ][ 25 9.420102 Ju-on__The_Grudge_(2003)   720p ][ 26 4.911898 Rocky Balboa    720p ][ 27 7.300815 Casablana.1942.1080p.BluRay.x264.Codres ][ 32 5.382049 Juno 2007 720p ][ 33 9.824923 Stargate-The Ark of Truth 720p ][ 35 1.239642 Van.Helsing  1080p ][ 36 8.716279 Final Fantasy VII Advent Children [2005] [1080p] ][ 37 7.403041 PIXAR Movie ][ 39 4.794048 Golden Compass   720p ][ 40 9.511793 Cleanwest 720p ][ 41 0.414628 Rocky 4 [1985] [BluRay] [720p] [AC3] ][ 42 4.765930 Interview with the Vampire   720p ][ 44 0.161225 Back.to.the.Future.Part.II.1989.720p.HDTV.x264 ][ 49 7.222352 Starship Troopers 2  720p ][ 50 1.446025 Tomb Raider Collection (1-2)    720p ][ 51 2.482692 Initial D [720p] ][ 54 7.474814 James Bond 21 - Casino Royale [2006] ][ 55 7.042460 Jeniffer's Body ( 2009)    1080p ][ 56 0.284166 Punisher Warzone [720p] ][ 59 0.841316 Full.Metal.Jacket.REMASTERED.1987.720p.HDDVD.x264-SEPTiC ][ 61 0.904714 Alien At The Attic 720P ][ 68 1.686811 Chicken Little 1080p ]
